pkg/spiffecert/svid.go

```diff
@@ -0,0 +1,186 @@
+/*
+Package spiffecert ...
+
+	When moving away from on-disk certificate configurations
+	and towards in memory certificate configurations some logic
+	in the disksvidwriter package was desired with much change
+	to its interface. Since that package only intends to provide
+	a means of writing certs to disk this package was made to use
+	in its place.
+*/
```

Comment:
From the open source perspective, this comment doesn't make much sense since we can't see the old packages you're referring to.